core/src/odbc_uri.rs

```diff
@@ -327,6 +332,20 @@ impl ODBCUri {
         let uuid_representation = ConnectionString::parse(uri)
             .map_err(Error::InvalidClientOptions)?
             .uuid_representation;
```

Comment:
Here is the part where we add the callback. Since MONGODB-OIDC can only be added via URI, we know there will be a URI